packages/block-editor/src/components/image-editor/use-save-image.js

```diff
@@ -71,16 +80,22 @@ export default function useSaveImage( {
 		const modifierType =
 			modifiers.length === 1 ? modifiers[ 0 ].type : 'cropAndRotate';
 
-		apiFetch( {
-			path: `/wp/v2/media/${ id }/edit`,
-			method: 'POST',
-			data: { src: url, modifiers },
-		} )
-			.then( ( response ) => {
+		try {
+			const savedImage = await editMediaEntity(
```

Comment:
Should we guard against undefined `editMediaEntity`? Which will be the case when the user has no upload permissions.